crates/holochain/src/conductor.rs

```diff
@@ -41,14 +41,13 @@ pub use handle::ConductorHandle;
 
 /// setup a tokio runtime that meets the conductor's needs
 pub fn tokio_runtime() -> tokio::runtime::Runtime {
-    tokio::runtime::Builder::new()
+    // we want to use multiple threads
+    tokio::runtime::Builder::new_multi_thread()
```

Comment:
Can we delete this whole function in favor of the tokio_helper static `TOKIO`? - We don't actually want more than one runtime - it'll just create extra threads to thrash the CPU...